src/workloads/execution/GroupSpillToDisk.yml

```diff
@@ -0,0 +1,126 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Runs $group queries which are designed to require spilling to disk.
+Keywords:
+- Loader
+- CrudActor
+- QuiesceActor
+- insert
+- aggregate
+- group
+- spill
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 2_000_000
+  FirstFieldUniqueValCount: &FirstFieldUniqueValCount 20
+  RandomStrLength: &RandomStrLength 60
+  MaxPhases: &MaxPhases 5
+
+Actors:
+# Phase 0: Insert documents into the collection.
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          firstField: {^RandomInt: {min: 1, max: *FirstFieldUniqueValCount}}
+          secondField: {^RandomString: {length: *RandomStrLength, alphabet: "0123456789"}}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# This query has a small number of groups, but each group is a very large set.
+- Name: AddToSetAccumulatorWithLargeSets
```

Comment:
[mega nit] prepend 'Group' to these tests ($addToSet, $push, $sum are also supported in $setWindowsFields)